CarRental/CarRental/CarRental.Domain/Data/CarRentalFixture.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CarRental.Domain_.Data;
+
+using CarRental.Domain_.Models;
+using global::CarRental.Domain_.Models;
+using System.Runtime.ConstrainedExecution;
+
+/// <summary>
+/// Fixture с тестовыми данными
+/// </summary>
+/// <summary>
+/// Fixture с тестовыми данными для ЛР1
+/// </summary>
+public class CarRentalFixture
+{
+    private readonly List<CarModel> _models;
+    private readonly List<ModelGeneration> _generations;
+    private readonly List<Car> _cars;
+    private readonly List<Customer> _customers;
+    private readonly List<RentalContract> _contracts;
+
+    public CarRentalFixture()
+    {
+        _models = new List<CarModel>
+        {
+            new() { ModelId = 1, Name = "Toyota Camry", DriveType = DriveType.FrontWheelDrive, SeatsCount = 5, BodyType = BodyType.Sedan, CarClass = CarClass.Middle },
+            new() { ModelId = 2, Name = "BMW X5",        DriveType = DriveType.AllWheelDrive,   SeatsCount = 7, BodyType = BodyType.SUV,   CarClass = CarClass.Premium },
+            new() { ModelId = 3, Name = "Volkswagen Golf", DriveType = DriveType.FrontWheelDrive, SeatsCount = 5, BodyType = BodyType.Hatchback, CarClass = CarClass.Economy },
+            new() { ModelId = 4, Name = "Audi A6",      DriveType = DriveType.AllWheelDrive,   SeatsCount = 5, BodyType = BodyType.Sedan, CarClass = CarClass.Premium },
+            new() { ModelId = 5, Name = "Mercedes C-Class", DriveType = DriveType.RearWheelDrive, SeatsCount = 5, BodyType = BodyType.Sedan, CarClass = CarClass.Premium }
+        };
```

Comment:
Упростить инициализацию коллекций.
```suggestion
        _models =
        [
            new() { ModelId = 1, Name = "Toyota Camry", DriveType = DriveType.FrontWheelDrive, SeatsCount = 5, BodyType = BodyType.Sedan, CarClass = CarClass.Middle },
            new() { ModelId = 2, Name = "BMW X5",        DriveType = DriveType.AllWheelDrive,   SeatsCount = 7, BodyType = BodyType.SUV,   CarClass = CarClass.Premium },
            new() { ModelId = 3, Name = "Volkswagen Golf", DriveType = DriveType.FrontWheelDrive, SeatsCount = 5, BodyType = BodyType.Hatchback, CarClass = CarClass.Economy },
            new() { ModelId = 4, Name = "Audi A6",      DriveType = DriveType.AllWheelDrive,   SeatsCount = 5, BodyType = BodyType.Sedan, CarClass = CarClass.Premium },
            new() { ModelId = 5, Name = "Mercedes C-Class", DriveType = DriveType.RearWheelDrive, SeatsCount = 5, BodyType = BodyType.Sedan, CarClass = CarClass.Premium }
        ];
```